tests/mail/tests.py

```diff
@@ -1552,6 +1552,25 @@ def test_all_params_can_be_set_before_send(self):
         self.assertIs(email.get_connection(), new_connection)
         self.assertNotIn("original", message.as_string())
 
+    def test_unicode_attachment_encoding(self):
+        """
+        Test that attaching a message containing non-ASCII characters does not
+        raise a UnicodeEncodeError (#36119)..
+        """
+        att = dedent(
+            """\
+            Content-Type: text/plain; charset=utf-8
+            Content-Transfer-Encoding: 8bit
+            ¡8-bit content!
+            """
```

Comment:
Oops, we actually need to keep the blank line between the "Content" headers and the content, to make it a valid "email message" attachment. (In the string serialization of an email message, a blank line is required to separate the headers from the body. We're trying to make `att` a minimal, but *valid,* email message that demonstrates the problem.)

```suggestion
            """\
            Content-Type: text/plain; charset=utf-8
            Content-Transfer-Encoding: 8bit

            ¡8-bit content!
            """
```

Reply:
Ok, have to #noqa the variable att so... otherwise linter cries

Reply:
Leave an empty line *without any spaces* on it, and flake8 should be OK. (It might help to adjust your editor to show whitespace.)

Reply:
looks like good